packages/block-editor/src/components/media-placeholder/content.scss

```diff
@@ -1,27 +1,11 @@
 .block-editor-media-placeholder__url-input-form {
-	display: flex;
-
-	// Selector requires a lot of specificity to override base styles.
-	input[type="url"].block-editor-media-placeholder__url-input-field {
```

Comment:
Interestingly, these styles were not applying to the `input` as intended because it didn't have `type="url"`.

To minimize the risk of unintended consequences for the scope of this PR, I didn't add a `type="url"` to the InputControl. Let me know if you disagree.